common/persistence/cassandraPersistence.go

```diff
@@ -1162,7 +1167,7 @@ func (d *cassandraPersistence) CreateWorkflowExecutionWithinBatch(request *Creat
 			permanentRunID,
 			defaultVisibilityTimestamp,
 			rowTypeExecutionTaskID,
-			request.PreviousRunID,
+			prevRunID,
```

Comment:
data type changed from string to *string, is this expected? why do we make this change?